### DISCUSSION TITLE
Build on jammy

### DISCUSSION THREAD

**Discussion by lrdbe** [AlexandreRouma/SDRPlusPlus#880] [Q&A] [answered]
I saw with pleasure that the .deb version had been made available on the nighty. How to recover the .deb file?
Thank you all.

**Comment by AlexandreRouma** [comment, marked as answer]
scroll down to artifact, click on the version you need, it's a zip

**Comment by lrdbe** [comment]
Wow, sorry, so simple!
Thank you very much and congratulations for this excellent software.

Le lun. 3 oct. 2022 à 17:01, AlexandreRouma ***@***.***> a
écrit :
